sdk/storage/storage-blob/src/policies/StorageRetryPolicy.ts

```diff
@@ -247,6 +247,21 @@ export class StorageRetryPolicy extends BaseRequestPolicy {
       }
     }
 
+    if (response) {
```

Comment:
Should we keep the code as the feature is removed from STG94. 
However, this code won't impact the function behavior when the feature not enabled in server, per my discussion with Emma.